src/providers/messages.js

```diff
@@ -1,25 +1,60 @@
-export const SUBSCRIBE = {
-    'yamcs.events': (subscriptionDetails) => {
-        return `{
-            "type": "events",
-            "id": "${subscriptionDetails.subscriptionId}",
-            "options": {
-                "instance": "${subscriptionDetails.instance}"
-            }
-          }`;
-    }
+import { OBJECT_TYPES, DATA_TYPES } from '../const';
+
+
+const typeMap = {
+    [OBJECT_TYPES.EVENTS_OBJECT_TYPE]: DATA_TYPES.DATA_TYPE_EVENTS,
```

Comment:
you mean `OBJECT_TYPES.EVENTS_OBJECT_TYPE: DATA_TYPES.DATA_TYPE_EVENTS,` 
instead of `[]`

Reply:
that's just to get the value as the key, other wise the key wouldn't match what I need it to